services/coderabbit/coderabbit-stats.service.js

```diff
@@ -0,0 +1,68 @@
+import Joi from 'joi'
+import { BaseJsonService, pathParams } from '../index.js'
+
+const schema = Joi.object({
+  reviews: Joi.number().required(),
+}).required()
+
+class CodeRabbitStats extends BaseJsonService {
+  static category = 'analysis'
+  static route = {
+    base: 'coderabbit',
+    pattern: 'prs/:provider/:org/:repo',
+  }
+
+  static openApi = {
+    '/coderabbit/prs/{provider}/{org}/{repo}': {
+      get: {
+        summary: 'CodeRabbit Pull Request Reviews',
+        description:
+          'This badge pulls the number of PRs reviewed by [CodeRabbit](https://coderabbit.ai), AI code review tool',
+        parameters: pathParams(
+          {
+            name: 'provider',
+            example: 'github, gitlab, bitbucket',
+            description: 'Version Control Provider (e.g., github)',
```

Comment:
The example value should be a value that works, so "github" would be sensible value for `example` here. Basically if I copy and paste it into the builder, it should work.

We also don't need to double up examples in the description.

In the docs, if we want to communicate to the user that there is a closed set of valid values, we can give the user a dropdown to pick from. So if you look at something like the `vcsType` param on https://shields.io/badges/coveralls you get a dropdown for [github, bitbucket, gitlab].

![Screenshot at 2024-12-23 18-12-35](https://github.com/user-attachments/assets/189ff8a3-1731-4530-b2e7-98fb7952477c)

Here's what that looks like in the code.

https://github.com/badges/shields/blob/0cbc1319ce30e1af262c88c12fb7e0deb890e6f3/services/coveralls/coveralls.service.js#L17

https://github.com/badges/shields/blob/0cbc1319ce30e1af262c88c12fb7e0deb890e6f3/services/coveralls/coveralls.service.js#L26-L30

Lets do the same here.

Reply:
I'm ok for the time-being it showing only GH. As GH has most usage, I'll be happy to update in the next revision with the upcoming badges. 

Editing the text to only show GitHub.

Reply:
OK. Are `gitlab` and `bitbucket` values that work today, or not?

If they are, lets make this

```js
static route = {
  base: 'coderabbit',
  pattern: 'prs/:provider(github|bitbucket|gitlab)/:org/:repo',
}
```

in the route and

```js
{
  name: 'provider',
  example: 'github',
  description: 'Version Control Provider',
  schema: { type: 'string', enum: this.getEnum('provider') },
}
```

in the Open API spec.

That will show it as

![Screenshot at 2024-12-23 20-21-57](https://github.com/user-attachments/assets/46eb9d4a-9a36-4293-b88e-b08a661ab542)

in the docs, and we're good to go.

Reply:
Made this change as you mentioned. Thanks for the suggestion!